.github/workflows/tox.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         lfs: 'true'
     - uses: actions/setup-python@v5
       with:
-        python-version: "3.10"
+        python-version: "3.11"
```

Comment:
What drove this change? I'm not seeing where we have this application pinned to 3.11 in the Dockerfile or Conda environment.

Reply:
It's derived from setup.cfg which should be updated as well.  This stemmed from a bad config earlier where we imported UTC from datetime (added in 3.11) but we only had tox running on 3.11, not 3.10 so we never caught it.  Instead of a larger code change to make the package compatible with 3.10 this just corrects it to accurately reflect what we have.